webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -1605,6 +1605,7 @@
                             owl:minCardinality "0"^^xsd:nonNegativeInteger ;
                             salsah-gui:guiOrder "1"^^xsd:nonNegativeInteger
                         ] ,
+
```

Comment:
You can get rid of this newline. I know that we are not consistent in formatting our Turtle ...